### ISSUE TITLE
Feature Request: option to see only active leases

### ISSUE THREAD

**Issue by beeforbacon** [lightninglabs/pool#206]
Requesting additional command for 
```pool auction leases``` 
where the output is only active leases rather than all the active + historical leases.

Perhaps ```pool auction leases --active-only``` ?

**Comment by guggero** [comment]
Sounds useful, yes.
Just to clarify: With "active" you mean leases where the channel is still open? Because if the lease is expired but the channel wasn't closed, it would still be considered to be active IMO.

**Comment by beeforbacon** [comment]
Good point. I was thinking leases where channel is still open whether or not the lease has expired is considered active. In the normal output of ```pool auction leases``` you can see the expiry height (although lately some leases show height as 0 but that's another topic) so it should be easy  at a glance know if the expiry height has been reached.
